tox.ini

```
@@ -0,0 +1,12 @@
+[tox]
+envlist = py38-{pycurl,requests,urlib},py37-{pycurl,requests,urlib},py36-{pycurl,requests,urlib}
```

We can make this list shorter and let's add another option: 
```suggestion
envlist = py{36,37,38}-{pycurl,requests,urlib}
skip_missing_interpreters=true
```